docs/search/get-started-with-search/suggested-searches/index.md

```diff
@@ -6,11 +6,40 @@ description: Sumo Logic queries can help track and diagnose common IT issues. Ta
 
 Sumo Logic queries can help track and diagnose common IT issues. Take a look at these Sumo Logic suggested searches.
 
-import DocCardList from '@theme/DocCardList';
-import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
 
 ## Guide contents
 
 In this section, we'll introduce the following concepts:
 
-<DocCardList items={useCurrentSidebarCategory().items}/>
+<div className="box-wrapper" markdown="1">
+<div className="box box1 card">
+  <div className="container">
+  <h4><a href="/docs/search/get-started-with-search/suggested-searches/apache-access-parser">Apache Access Parser</a></h4>
+  <p>The following searches were built for use with the Apache Access Parser.</p>
```

Comment:
These descriptions need to make sense as standalone statements on a separate page. You could say something like "Learn about suggested searches for the Apache Access Parser".

Reply:
I did not write it on my own. It has been there in the system for a long I guess. But yes, we should change them.

Reply:
Is this tied to an urgent feature release? Apologies if I misunderstood; I thought this was my Hub Pages project.

To clarify, as we're building out our assigned Hub Pages (I thought that's what this PR was) I didn't mean to pull descriptions from the `description` field of that particular doc. I've written 90% of them from scratch.

Reply:
@kimsauce Thank you so much for your contribution, you're doing awesome. For this doc, we should be good now as I have descriptions.